src/DocChecks.jl

```diff
@@ -180,17 +180,21 @@ function linkcheck(link::Markdown.Link, doc::Documents.Document)
     end
 
     if !haskey(doc.internal.locallinks, link)
+        cmd = `curl -sI --proto =http,https,ftp,ftps $(link.url) --max-time 10`
```

Comment:
Is the space between `--proto` and `=` correct / necessary? Interestingly, it doesn't seem to affect the command.

Reply:
It is necessary, as it denies all other protocols. 

```
       --proto <protocols>
              Tells curl to limit what protocols it may use in the transfer. Protocols are evaluated left  to  right,  are  comma
              separated, and are each a protocol name or

              +  Permit this protocol in addition to protocols already permitted (this is the default if no modifier is used).

              -  Deny this protocol, removing it from the list of protocols already permitted.

              =  Permit  only this protocol (ignoring the list already permitted), though subject to later modification by subse-
                 quent entries in the comma separated list.

              For example:

              --proto -ftps  uses the default protocols, but disables ftps

              --proto -all,https,+http
                             only enables http and https

              --proto =http,https
                             also only enables http and https

       Unknown protocols produce a warning. This allows scripts to safely rely on being able  to  disable  potentially  dangerous
       protocols, without relying upon support for that protocol being built into curl to avoid an error.

       This  option  can  be  used  multiple  times, in which case the effect is the same as concatenating the protocols into one
       instance of the option.
```

Reply:
Ah, ok, makes sense. Thanks!